stylus/objects/layouts.styl

```diff
@@ -82,15 +96,7 @@ $app-2panes-sticky
         height auto
 
     +medium-screen()
-        display block
-
-        main,
-        main > [role=contentinfo]
-            display block
-            overflow visible
-
-        // Those pseudo-elements are "ghost" elements replacing bar and nav
-        &:before
+        // This pseudo-element is a "ghost" element replacing nav
```

Comment:
Extra comment here?

Reply:
Well not quite the same as the one before but it does means the same.